vagrant_opm_user/bootstrap.sh

```diff
@@ -20,9 +20,9 @@ sudo apt-get update -y
 
 # OPM packages
 sudo apt-get install libopm-core1-bin -y
-sudo apt-get install libopm-autodiff1-bin -y
-sudo apt-get install libopm-polymer1-bin -y
+sudo apt-get install libopm-simulators1-bin -y
 sudo apt-get install libopm-upscaling1-bin -y
+sudo apt-get install openmpi-bin
```

Comment:
> if you think convenience trumps being correct i can have the opm-simulators-bin depend on mpi-default-bin in the future

I think it sounds like a good idea, since otherwise no-one can run the program... Also: while I can see how forcing a specific MPI implementation is not Debian-like and incorrect, depending on mpi-default-bin would not make that error, so why is that incorrect in your opinion?